README.md

```
@@ -1,128 +1,310 @@
+npm test
+
+# Run tests with coverage
+npm run test:coverage
 ```
 
-4. Start the server:
-```bash
-npm start
+### Test Coverage
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)